packages/commonwealth/server/api/external-router.ts

```diff
@@ -109,6 +114,19 @@ const api = {
   createContestMetadata,
   updateContestMetadata,
   cancelContestMetadata,
+  deleteContestMetadata,
+  getContestLog: trpc.query(Contest.GetContestLog, trpc.Tag.Contest, {
+    forceSecure: true,
+  }),
+  getFarcasterCasts: trpc.query(
```

Comment:
This also seems dangerous because we use the rate-limited Neynar API within the query, so this basically allows any API user to use our Neynar keys.